eng/scripts/Language-Settings.ps1

```diff
@@ -130,3 +130,17 @@ function SetPackageVersion ($PackageName, $Version, $ReleaseDate, $PackageProper
     -ReleaseDate $ReleaseDate `
     -ReplaceLatestEntryTitle $ReplaceLatestEntryTitle
 }
+
+
+function Find-Go-Artifacts-For-Apireview($ArtifactPath, $PackageName)
+{
+  $artifact = Get-ChildItem -Path (Join-Path $ArtifactPath $PackageName) -Filter "*.gosource"
```

Comment:
Does this need `-Recurse` for nested paths or no?

Reply:
Not really. I will be surprised if it happens to be in another sub directory and we can take a look.